packages/wdio-mocha-framework/tests/adapter.test.ts

```diff
@@ -181,7 +181,14 @@ test('prepareMessage', async () => {
     expect(result.type).toBe('afterTest')
     expect(result.title).toBe('foobar')
     expect(result.file).toBe('/foo/bar.test.js')
-})
+
+    await new Promise((r) => setTimeout(r, 5000))
+    result = adapter.prepareMessage('afterSuite')
+    expect(result.type).toBe('afterSuite')
+    expect(result.title).toBe('first suite')
+    expect(result.duration).toBeDefined()
+    expect(result.duration >= 5000 && result.duration <= 5020).toBeTruthy()
+}, 6000)
```

Comment:
@christian-bromann 

Hi the test is infered as Mocha.test but it is jest test so the 6000 will show error in the vscode . Do you want to add an explicit import for jest as 

`import {describe, expect, test} from '@jest/globals'.`

or add a type for test as

`let test:jest.It`

or just ignore the error

![image](https://user-images.githubusercontent.com/45095911/153182334-58daac5f-945c-413d-9e95-68d4f80fbd5d.png)